README.md

```diff
@@ -7,7 +7,8 @@ This subproject hosts a HAPI FHIR server that is based on the [hapi-fhir-jpaserv
 3. from Terminal (2) while in test-ehr folder:   `gradle loadData`
 
 ## Running the server
-`gradle bootRun`
+1. Make sure `gradle` is installed on your machine. Gradle v8 or higher.
+2. Run `gradle bootRun`
```

Comment:
This is repeated from the section above. I would suggest we remove the section above and keep this one. Then add another section for cleanup that lists the above steps, perhaps later on in the README.